### PR TITLE
Wishlist & Cart Management

### PR DESCRIPTION
**Pull request by ark2002**
- Added wishlist page
- Added cart management  page
- linked all the new pages
- Wishlist page : https://deploy-preview-4--astromart-screens.netlify.app/wishlistpage/wishlistpage.html
- Cart Management page : https://deploy-preview-4--astromart-screens.netlify.app/cartmanagement/cartpage.html

### CODE REVIEW

**Review by MarkVed17 — APPROVED** [review]
LGTM!

File: cartmanagement/cartpage.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    padding-bottom: 1rem;
+}
+
+.price__summary:nth-child(4) {
+    border-bottom: 1px solid #999999;
+}
+
+.price__summary:nth-child(5) {
+    border-bottom: 1px solid #999999;
+}
```
Comment (horsemaker) [inline cartmanagement/cartpage.css:98]: You could use `hr` tags instead of this.

File: wishlistpage/wishlistpage.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    justify-content: space-around;
+    flex-basis: 20%;
+}
+
+.wishlist__selection {
+    background-color: var(--white-color);
+    border: none;
+}
+
+.wishlist__selection>.material-icons {
```
Comment (horsemaker) [inline wishlistpage/wishlistpage.css:39]: I guess we aren't supposed to alter the material icon's class directly. Instead, you could add your own class & make changes through that.

**Review by shlok30 — APPROVED** [review]
Looks good to merge 
Just 1 suggestion, add hover effect to your CTAs (maybe darken the purple colour on hover)

**Review by horsemaker — APPROVED** [review]
The design & theme is dope! 👌
The Obsidian Pyramid is 🤌❤️

An excellent naming convention is followed in the CSS files. 👍

File: cartmanagement/cartpage.css (hunk 1 above)
Comment (amirnathani9) [inline cartmanagement/cartpage.css:98]: give the variable name to color and use it. if using only here write a small comment

Comment (ark2002) [reply cartmanagement/cartpage.css:98]: i would keep that in mind

File: cartmanagement/cartpage.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Cart | AstroMart</title>
+    <link rel="shortcut icon" href="/assets/favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="/common.css">
+    <link rel="stylesheet" href="/cartmanagement/cartpage.css" />
+    <link rel="stylesheet" href="https://astro-ui.netlify.app/main.css">
+    <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css" />
```
Comment (amirnathani9) [inline cartmanagement/cartpage.html:13]: use relative path notation `<link rel="stylesheet" href="./common.css">` instead of normal notation

**Review by amirnathani9 — APPROVED** [review]
All good. Do the suggested things

**Pull request merged**